EZSwiftExtensionsTests/UILabelTests.swift

```diff
@@ -7,49 +7,66 @@
 //
 
 #if os(iOS) || os(tvOS)
-
-import XCTest
-@testable import EZSwiftExtensions
-class UILabelTests: XCTestCase {
-
-    func testInit() {
+    
+    import XCTest
+    @testable import EZSwiftExtensions
+    class UILabelTests: XCTestCase {
         
-        let label = UILabel(x: 0, y: 0, w: 200, h: 50)
```

Comment:
Are there any changes in these parts? Please make the code change idempotent to only the changes you are making. This looks like an indentation change which is unnecessary. The code already was properly indented.